### PR TITLE
UI #2567 donor summary table

### PR DESCRIPTION
**Pull request by samrichca**
- add filters, tabs, and pagination to tables

platform UI PR https://github.com/icgc-argo/platform-ui/pull/2635

### CODE REVIEW

File: src/TableV8/stories.tsx [hunk 1]
```diff
@@ -17,15 +17,37 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (samrichca) [inline src/TableV8/stories.tsx:17]: stories are WIP - I'll improve them if everything else is OK

File: src/TableV8/TableFilters/index.tsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
+/*
```
Comment (samrichca) [inline src/TableV8/TableFilters/index.tsx:1]: moved filter components from platform-UI to uikit so they can be reused

File: src/TableV8/TableCells.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/*
```
[the remark this answers is not in the document]
Comment (demariadaniel) [reply src/TableV8/TableCells.tsx:1]: Any updates? Can always do a follow up ticket

[the remark this answers is not in the document]
Comment (samrichca) [reply src/TableV8/TableCells.tsx:1]: not yet, I've been finding other custom cells to be very tightly related to logic & types in their respective tables. it was a nice idea though.

File: src/TableV8/TableFilters/index.tsx (hunk 2 above)
Comment (demariadaniel) [reply src/TableV8/TableFilters/index.tsx:1]: Tested + working with local Summary table

File: src/TableV8/stories.tsx (hunk 1 above)
Comment (demariadaniel) [reply src/TableV8/stories.tsx:17]: Follow up to Slack convo, we should make story follow up tickets to pick up when time allows

**Review by demariadaniel — APPROVED** [review]

**Pull request merged**